spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/oauth2/resource/servlet/OAuth2ResourceServerJwkConfiguration.java

```diff
@@ -40,9 +43,17 @@
 	@Bean
 	@ConditionalOnProperty(name = "spring.security.oauth2.resource.jwt.jwk.set-uri")
 	@ConditionalOnMissingBean
-	public JwtDecoder jwtDecoder() {
+	public JwtDecoder jwtDecoderByUri() {
 		return new NimbusJwtDecoderJwkSupport(
 				this.properties.getJwt().getJwk().getSetUri());
 	}
 
+	@Bean
+	@ConditionalOnProperty(name = "spring.security.oauth2.resource.jwt.jwk.oidc-issuer-location")
```

Comment:
The property should not use the `jwk` group as this configuration isn't related to it. The property should be more like:

`spring.security.oauth2.resource.jwt.oidc-issuer-location`

Reply:
yeah, my mistake, I've fixed it